src/pykeen/pipeline/api.py

```diff
@@ -584,8 +584,15 @@ def pipeline_from_config(
 
 def _build_model_helper(
     *,
-    model, model_kwargs, loss, loss_kwargs, _device, _random_seed, regularizer, regularizer_kwargs,
-    training,
+    model,
+    model_kwargs,
+    loss,
+    loss_kwargs,
+    _device,
+    _random_seed,
+    regularizer,
+    regularizer_kwargs,
+    training_triples_factory,
```

Comment:
type annotation?

Reply:
@cthoyt Coming from https://github.com/pykeen/pykeen/pull/326

Reply:
Is a private function anyway